contracts/modules/governance/VotePower.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+
+import "../../interfaces/IGovernance.sol";
+import "../../interfaces/IMemberRoles.sol";
+import "../../interfaces/INXMMaster.sol";
+import "../../interfaces/INXMToken.sol";
+import "../../interfaces/ITokenController.sol";
+
+contract VotePower {
+
+  string constant public name = "NXM balance with delegations";
+  string constant public symbol = "NXMD";
+  uint8 constant public decimals = 18;
+
+  INXMMaster immutable public master;
+
+  enum Role {UnAssigned, AdvisoryBoard, Member}
+
+  constructor(INXMMaster _master) {
+    master = _master;
+  }
+
+  function totalSupply() public view returns (uint) {
+    return INXMToken(master.tokenAddress()).totalSupply();
+  }
+
+  function balanceOf(address member) public view returns (uint) {
+
+    ITokenController tokenController = ITokenController(master.dAppLocker());
+    IMemberRoles memberRoles = IMemberRoles(master.getLatestAddress("MR"));
+    IGovernance governance = IGovernance(master.getLatestAddress("GV"));
+
+    uint delegationId = governance.followerDelegation(member);
+
+    if (delegationId != 0) {
+      (, address leader,) = governance.allDelegation(delegationId);
+      if (leader != address(0)) {
+        return 0;
+      }
+    }
+
+    uint balance = tokenController.totalBalanceOf(member) + 1e18;
```

Comment:
what is the reason for the `+ 1e18` ?

Reply:
Required by the formula: each wallet votes with amount of nxm held + 1 nxm. In other words it's not a tech-related requirement but a governance rules one.